nextmv/nextmv/cloud/manifest.py

```diff
@@ -481,8 +530,44 @@ def to_option(self) -> Option:
             description=self.description,
             required=self.required,
             additional_attributes=self.additional_attributes,
+            control_type=self.ui.control_type if self.ui else None,
+            hidden_from=self.ui.hidden_from if self.ui else None,
         )
 
+class ManifestValidation(BaseModel):
+    """
+    Validation rules for options in the manifest.
+
+    You can import the `ManifestValidation` class directly from `cloud`:
+
+    ```python
+    from nextmv.cloud import ManifestValidation
+    ```
+
+    Parameters
+    ----------
+    enforce : str, default="none
```

Comment:
```suggestion
    enforce : str, default="none"
```

Just a missing quote, I think